_plugins/idp_translations.rb

```diff
@@ -0,0 +1,34 @@
+## This plugin copies the fonts and the images from the identity style guide
+## to the appropriate path on the built site
+##
+## The CSS and JS is already handled by Jekyll and the build:js npm script
+
+require 'fileutils'
+require 'uri'
+require 'net/http'
+require 'json'
+require 'open-uri'
+
+DOMAIN = 'https://secure.login.gov'
+
+def copy_translations(origin_uris, destination)
+  origin_uris.each do |uri|
+    filename = uri.gsub(DOMAIN, '').gsub(%r{^/}, '')
+    dest_filename = File.join(destination, filename)
+    next if File.exist?(dest_filename)
+
+    download = URI.open("#{DOMAIN}#{uri}")
```

Comment:
oops we are using `URI.open` here scratch previous comment https://github.com/18F/identity-site/pull/1150/files#r1309291788